pom.xml

```diff
@@ -990,12 +1015,48 @@
                                 <version>${mtf-tools.version}</version>
                                 <classifier>mule-plugin</classifier>
                             </dependency>
+                            <dependency>
+                                <groupId>org.apache.commons</groupId>
+                                <artifactId>commons-dbcp2</artifactId>
+                                <version>2.9.0</version>
+                            </dependency>
                             <dependency>
                                 <groupId>org.mule.connectors</groupId>
                                 <artifactId>mule-vm-connector</artifactId>
                                 <version>${mule.vm.version}</version>
                                 <classifier>mule-plugin</classifier>
                             </dependency>
+                            <dependency>
+                                <groupId>org.mule.modules</groupId>
+                                <artifactId>mule-spring-module</artifactId>
+                                <version>${mule-spring-module-version}</version>
+                                <classifier>mule-plugin</classifier>
+                            </dependency>
+                            <dependency>
+                                <groupId>org.springframework</groupId>
```

Comment:
Are these spring dependencies supposed to be test scope?

Reply:
Hello @sabecasismulesoft  yes is just for testing purpose